src/main/kotlin/com/ampnet/walletservice/grpc/blockchain/BlockchainServiceImpl.kt

```diff
@@ -444,7 +444,7 @@ class BlockchainServiceImpl(
         val errorCode = ErrorCode.INT_GRPC_BLOCKCHAIN
         errorCode.specificCode = grpcErrorCode.code
         errorCode.message = grpcErrorCode.message
-        return GrpcException(errorCode, message)
+        return GrpcException(errorCode, message, ex)
     }
 
     // Status defined in ampenet-blockchain service, for more info see:
```

Comment:
If statement more concise:
```suggestion
private fun getErrorDescriptionFromExceptionStatus(ex: StatusRuntimeException): GrpcErrorCode {
        val description = ex.status.description?.split(" > ") ?: throw ex
        if (description.size != 2) throw ex
        return GrpcErrorCode(description[0], description[1])
    }
```